src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl

```diff
@@ -73,3 +79,31 @@ const GFBIBG = ImmersedBoundaryGrid{<:Any, <:Any, <:Any, <:Any, <:Any, <:GridFit
 
 Adapt.adapt_structure(to, ib::GridFittedBottom) = GridFittedBottom(adapt(to, ib.bottom))     
 
+#####
+##### Implicit vertical diffusion
+#####
+
+# Tracers and horizontal velocities at cell centers in z
+
+@inline function ivd_upper_diagonalᵃᵃᶜ(i, j, k, ibg::GFIBG, clock, Δt, κ⁻⁻ᶠ, κ)
```

Comment:
there is a problem here since GFIBG is not defined. I tried to use GFBIBG instead but still getting an error that seems to be related to changes in this file.

Reply:
Sorry about that, I'll fix it.